PKGBUILD

```diff
@@ -4,10 +4,9 @@ pkgdesc="ROS - rqt_gui provides the main to start an instance of the ROS integra
 url='https://wiki.ros.org/rqt_gui'
 
 pkgname='ros-noetic-rqt-gui'
-pkgver='0.5.0'
-_pkgver_patch=0
+pkgver='0.5.2'
 arch=('any')
-pkgrel=2
+pkgrel=3
```

Comment:
I guess I missed this in the last review mb. Whenever we update the pkgver we reset the pkgrel down to 1. See: https://wiki.archlinux.org/index.php/PKGBUILD#pkgrel